zulipterminal/model.py

```diff
@@ -485,7 +486,9 @@ def append_message(self, event: Event) -> None:
         Adds message to the end of the view.
         """
         response = event['message']
-        response['flags'] = []
+        # sometimes `flags` are missing in `event` so initialize
+        # an empty list of flags in that case.
+        response['flags'] = [] if not event.get('flags') else event['flags']
```

Comment:
I think this can be `event.get('flags', [])` ? That seems clearer?

Reply:
oh! I forgot we can do that.